templates/link_template.function_app.configuration.json

```diff
@@ -117,7 +125,8 @@
     },
     "variables": {
         "enableFortiAnalyzer": "[equals(parameters('FortiAnalyzerIntegrationOptions'), 'yes')]",
-        "funcappURL": "[concat('https://', parameters('FunctionAppName'), '.azurewebsites.net')]"
+        "funcappURL": "[concat('https://', parameters('FunctionAppName'), '.azurewebsites.net')]",
+        "terminateUnhealthyVM": "[equals(parameters('TerminateUnhealthyVM'), 'yes')]"
```

Comment:
if it is changed here in: https://github.com/fortinet/fortigate-autoscale-azure/pull/74/files#diff-7a23b18a77e3e1aacaa7f76e333107896fa16a72aa5addb70c0f6354a2f35614R419 we do not need this line
```suggestion
```